docs/data/pages.ts

```diff
@@ -31,7 +31,27 @@ const pages: MuiPage[] = [
           { pathname: '/x/react-data-grid/column-groups', title: 'Column groups 🚧' },
         ],
       },
-      { pathname: '/x/react-data-grid/rows' },
+      {
+        pathname: '/x/react-data-grid/rows',
+        scopePathnames: [
+          '/x/react-data-grid/row-definition',
+          '/x/react-data-grid/row-updates',
+          '/x/react-data-grid/row-dimensions',
+          '/x/react-data-grid/row-ordering',
+          '/x/react-data-grid/master-detail',
+          '/x/react-data-grid/row-pinning',
+          '/x/react-data-grid/row-spanning',
+        ],
+        children: [
+          { pathname: '/x/react-data-grid/row-definition' },
+          { pathname: '/x/react-data-grid/row-updates' },
+          { pathname: '/x/react-data-grid/row-dimensions' },
+          { pathname: '/x/react-data-grid/row-ordering', plan: 'pro' },
+          { pathname: '/x/react-data-grid/master-detail', plan: 'pro' },
+          { pathname: '/x/react-data-grid/row-pinning', plan: 'pro', title: 'Row pinning 🚧' },
+          { pathname: '/x/react-data-grid/row-spanning', title: 'Row spanning 🚧' },
```

Comment:
"master detail" in the middle of "rows" breaks the flow.

```suggestion
          { pathname: '/x/react-data-grid/row-pinning', plan: 'pro', title: 'Row pinning 🚧' },
          { pathname: '/x/react-data-grid/row-spanning', title: 'Row spanning 🚧' },
          { pathname: '/x/react-data-grid/master-detail', plan: 'pro' },
```

Reply:
I think we can see it in two ways

1. We put *Master Detail* quite high because it's an important feature compared to some existing or upcoming row features

2. We put *Master Detail* at the end because it's the only one that does not start with *Row*


@samuelsycamore

Reply:
@joserodolfofreitas as well

Reply:
I think it's enough if `Master detail` is the first Pro-listed feature. The break on the pattern name (+ Pro indication) will probably be enough to grab some extra attention.

Reply:
I've moved master detail to be the first Pro feature in the navbar